modules/099-ceph-csi/crds/doc-ru-cephcsi.yaml

```diff
@@ -0,0 +1,119 @@
+                rbd:
+                  type: object
+                  properties:
+                    storageClasses:
+                      description: |
+                        Описание StorageClass'ов для Rados Block Device (RBD)
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          name:
+                            description: |
+                              Имя StorageClass
+                            type: string
+                          pool:
+                            description: |
+                              Название пула в котором будут создаваться RBD-образы
+                            type: string
+                          reclaimPolicy:
+                            description: |
+                              Политика возврата для PersistentVolume
+                              https://kubernetes.io/docs/concepts/storage/persistent-volumes/#reclaiming
+                            type: string
+                          allowVolumeExpansion:
+                            description: |
+                              Включает возможность изменять размер тома, редактируя соответствующий объект PersistentVolumeClaim
+                              https://kubernetes.io/docs/concepts/storage/storage-classes/#allow-volume-expansion
+                            type: boolean
+                          mountOptions:
+                            description: |
+                              Список опций монтирования
+                            type: array
+                            items:
+                              type: string
+                          defaultFSType:
+                            description: |
+                              Файловая система по умолчанию для создаваемых Persistent Volumes
+                            type: string
+                            enum:
+                              - ext4
+                              - xfs
+                cephfs:
+                  type: object
+                  properties:
+                    subvolumeGroup:
+                      description: |
+                        Имя группы для subvolumes
+                      type: string
+                    storageClasses:
+                      description: |
+                        Описание StorageClass'ов для CephFS
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          name:
+                            description: |
+                              Имя StorageClass
+                            type: string
+                          pool:
+                            description: |
+                              Название пула в котором будут создаваться RBD-образы
+                            type: string
+                          reclaimPolicy:
```

Comment:
If the reclaim policy is changed, we do not have a hook to delete the storage class. Helm upgrade will fail in this case.

Reply:
Removing storage class added to get_crds.